deployments/README.md

```diff
@@ -79,8 +79,49 @@ not strictly required.
 
 ## General Notes
 
-Substrate authorities are named: `Alice`, `Bob`, `Charlie`, `Dave`, `Eve`, `Ferdie`.
-Ethereum authorities are named: `Arthur`, `Bertha`, `Carlos`.
+Rialto authorities are named: `Alice`, `Bob`, `Charlie`, `Dave`, `Eve`.
+Rialto-PoA authorities are named: `Arthur`, `Bertha`, `Carlos`.
+Millau authorities are named: `Alice`, `Bob`, `Charlie`, `Dave`, `Eve`.
+
+Both authorities and following accounts have enough funds on corresponding Substrate chains:
+
+- on Rialto: `Ferdie`, `George`, `Harry`.
+- on Millau: `Ferdie`, `George`, `Harry`.
+
+Names of accounts on Substrate (Rialto and Millau) chains may be prefixed with `//` and used as
+seeds for the `sr25519` keys. This seed may also be used in the signer argument in substrate
+and PoA relays. Example:
+
+```bash
+./substrate-relay rialto-headers-to-millau \
+	--rialto-host rialto-node-alice \
+	--rialto-port 9944 \
+	--millau-host millau-node-alexander \
+	--millau-port 9944 \
+	--rialto-signer //Harry \
+	--prometheus-host=0.0.0.0
+```
+
+Some accounts are used by bridge components. Using these accounts to sign other transactions
+is not recommended, because this may lead to nonces conflict.
+
+Following accounts are used when `poa-rialto` bridge is running:
+
+- Rialto' `Alice` signs relay transactions with new Rialto-PoA headers;
+- Rialto' `Bob` signs relay transactions with Rialto-PoA -> Rialto currency exchange proofs.
+- Rialto-PoA' `Arthur`: signs relay transactions with new Rialto headers;
+- Rialto-PoA' `Bertha`: signs currency exchange transactions.
+
+Following accounts are used when `rialto-millau` bridge is running:
+
+- Millau' `Charlie` signs relay transactions with new Rialto headers;
+- Rialto' `Charlie` signs relay transactions with new Millau headers;
+- Millau' `Dave` signs Millau transactions with messages to Rialto;
+- Rialto' `Dave` signs Rialto transactions with messages to Millau;
+- Millau' `Eve` signs relay transactions with message delivery confirmations from Rialto to Millau;
+- Rialto' `Eve` signs relay transactions with messages from Millau to Rialto;
```

Comment:
I'm not sure I understand what `Rialto's Eve' does. How does that differ from `Dave`?

Reply:
So Dave is used by the relayer that _submits_ a message that a user wants sent across a lane, while Eve is used for the relayer which servers (so actually sends) that message lane. This should be clarified a bit in the docs

Reply:
That's correct - Dave is signing transactions with new messages on the sending side. And Eve is used by relay to sign transactions that deliver these messages to the receiving side. Please suggest changes

Reply:
I left some suggestion. I'd use those/something similar with Dave, Eve, and Ferdie